client/components/organisms/PageLayout/PageLayout.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import treesIcon from '../../../assets/icons/trees.svg';
+import menuIcon from '../../../assets/icons/icon_menu.svg';
+
+import Container from 'react-bulma-companion/lib/Container';
+
+export default function PageLayout(props) {
```

Comment:
Great work, now, make sure the template wraps every other page in our app, excpet for the welcome page. (See main.js where all the routes are defined)